### DISCUSSION TITLE
Zan, Zendegi, Azadi - Logo (Vector)

### DISCUSSION THREAD

**Discussion by thisisnobodyx** [WeAreMahsaAmini/Core#304] [Ideas]
![Zan-Zendegi-Azadi (Vector)-Color](https://user-images.githubusercontent.com/41937317/193102684-039af185-985d-4d61-b46d-48ad81055d74.svg)
![Zan-Zendegi-Azadi (Vector)](https://user-images.githubusercontent.com/41937317/193102697-7985f7a8-63c9-45ab-af34-545431594877.svg)

**Comment by 0xland** [comment]
Nice, Could you please make the flag vector too?
